src/Opaleye/Internal/PGTypesExternal.hs

```diff
@@ -106,6 +107,10 @@ pgTimeOfDay = IPT.unsafePgFormatTime "time" "'%T%Q'"
 -- "We recommend not using the type time with time zone"
 -- http://www.postgresql.org/docs/8.3/static/datatype-datetime.html
 
+pgInterval :: Time.CalendarDiffTime -> Column PGInterval
+pgInterval = IPT.castToType "interval" . quote . Time.Format.ISO8601.iso8601Show
+    where
+      quote s = "'" ++ s ++ "'"
```

Comment:
Probably best just to call this `sqlInterval`. The `pg...` naming is deprecated, and we don't need to continue to use it, even in internal modules.